mods/d2k/sequences/misc.yaml

```diff
@@ -334,19 +334,20 @@ crate:
 		ZOffset: -511
 		Offset: -16,-16
 
-# TODO: keep the redundant spicebloom.shp for now for the awful WinForms maps editor
```

Comment:
Perhaps with the removal of this comment (and the long-gone WinForms map editor) you should remove that SHP?